src/module/apps/compendium-browser/tabs/feat.ts

```diff
@@ -133,6 +137,21 @@ export class CompendiumBrowserFeatTab extends CompendiumBrowserTab {
         console.debug("PF2e System | Compendium Browser | Finished loading feats");
     }
 
+    protected override filterTraits(
+        traits: string[],
+        selected: MultiselectData["selected"],
+        condition: MultiselectData["conjunction"]
+    ): boolean {
+        // Always return true for ancestry feats that have no ancestry trait
+        if (
+            this.filterData.checkboxes.category.selected.includes("ancestry") &&
+            !this.arrayIncludes(traits, this.#ancestryTraits)
+        ) {
+            return true;
```

Comment:
This works for the case where the selected traits are _all_ ancestry traits, but returns excess results if the user adds other non-ancestry traits because it returns early before checking any additional logic. 

For example, filtering on `["kobold", "mental"]` should return **Cringe** (matching kobold feat) and **Drain Emotion** (matching all-ancestries feat) but not other all-ancestry feats.

I'd suggest that instead of returning true here, you pass the selected trait list _with the ancestry traits removed_ to the super call so that you don't have to rebuild any logic here.

Reply:
Good catch. I agree that filtering the ancestry traits first should yield the expected result. I'll push up a new fix for my fix when I'm home.